new/resource-current-value.md

```diff
@@ -0,0 +1,98 @@
+---
+RFC: unassigned
+Author: John Keiser <jkeiser@chef.io
+Status: Draft
+Type: Standards Track
+---
+
+# Allow Current Value To Be Read From Resource
```

Comment:
Just current_value--which corresponds to the new `load_current_value` API. current_resource might be the right word, but in that case it would still correspond to `Provider.load_current_resource`, an existing API which lets you load the current value of the resource from the system.